security-policies/bundle/compliance/cis_azure/rules/cis_4_1_2/rule.rego

```diff
@@ -15,6 +16,20 @@ finding = result if {
 	)
 }
 
+default sql_access_config_is_permissive = false
+
+sql_access_config_is_permissive if {
+	lower(data_adapter.properties.publicNetworkAccess) == "enabled"
+}
+
+sql_access_config_is_permissive if {
+	_ = data_adapter.resource.extension.sqlFirewallRules
```

Comment:
Do you need to do `_ =` to filter? I thought that case it's undefined it already filters it out